kubefwd/sudo-kubefwd.sh

```diff
@@ -27,6 +29,32 @@ if [[ "$ENTR" == "" ]]; then
     exit 1
 fi
 
+# Initialize the namespaces we need to watch.
+# Write them to the file that we use to trigger reloads.
+NAMESPACES=$(tilt get kd -o=jsonpath='{.items[*].spec.watches[*].namespace}' | tr -s ' ' '\n' | sort -u)
+echo "$NAMESPACES" > "$TRIGGER"
```

Comment:
ffti - if we delete this does watch-namespaces.sh just do this anyway? (we probably still want to touch/chmod, or put the chmod in watch-namespaces.sh as well)